Source/Scene/Model.js

```diff
@@ -517,6 +521,24 @@ define([
         this._shadows = this.shadows;
 
         /**
+         * A color that blends with the model's rendered color.
+         *
+         * @type {Color}
+         *
+         * @default Color.RED
+         */
+        this.blendColor = defaultValue(options.blendColor, Color.RED);
```

Comment:
Is the intention to allow RGBA values?  If so, the render state may need to change on the fly when alpha changes from one to non-one.  Likewise, you may want to optimize the case when alpha is zero and not issue the `DrawCommand`.

If the intention is only RGB, the explicitly document that and throw when alpha is non-zero.

I would support alpha so users can fade in/out a model with this.